db/migrate/20171113163731_update_emails_reference_from_notifications_to_content_changes.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateEmailsReferenceFromNotificationsToContentChanges < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :emails, :notification_id
+    add_reference(:emails, :content_change, null: false, foreign_key: true)
```

Comment:
I'm not sure we actually need this new column back! I think we decided to not have a relation between emails and content changes.

Reply:
If we decided to keep this in, we'll run into a problem where all the fields will be `NULL` when the column is created, and that will fail because of the `null: false`. I wonder if we can remove the foreign key, rename the column and then bring it back.